strawberryfields/backends/tfbackend/tests/test_symbolic.py

```diff
@@ -0,0 +1,601 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Tests for the various Tensorflow-specific symbolic options of the frontend/backend.
+"""
+ #pylint: disable=expression-not-assigned,too-many-public-methods,pointless-statement
+
+import pytest
+
+import numpy as np
+from scipy.special import factorial
+import tensorflow as tf
+
+from strawberryfields.ops import Dgate, MeasureX
+
+ALPHA = 0.5
+
+
+def coherent_state(alpha, cutoff):
```

Comment:
alpha argument not used - not sure if this is a typo or not. Either it should be removed and just use the global variable, or change the function to use `alpha` as a parameter.

Reply:
good catch, fixed